### ISSUE TITLE
[0781 PS] Start Page Skeleton

### ISSUE THREAD

**Issue by SamStuckey** [department-of-veterans-affairs/vets-api#19390]
## Status

Estimate: L

Workability: blocked by 'Choice Page Skeleton'

Testability: partial skeleton

## Description

**OUT OF SCOPE:** Routing based on form state will be handled by 'Completion Choice and Top Level routing Skeleton'

Entry Point: 3.1*

Exit Point: 3.2, Opt Out Modal / ??? (if someone opts out, where do they go after the modal?)

This page reads the status of the users form from the data solution and presents the Form Tile states

- option to start (3.2)
- option to continue (3.2)
- option to edit (if complete) (TBD? start of form or summary?)

Additionally, The 'Continue' Button now has conditional behavior.

- IF a user has opted out
  - THEN show them the opt out confirmation modal
  - modal routs to TBD (where do they go after they opt out?)
- ELSE IF a user has an incomplete form (not opted out is implicit here)
  - THEY they are prompted to complete their form
- ELSE
  - THEN they are routed to the next page (TBD, what comes after??)
  - NOTE: this will be a stubbed out condition. Final realization of this will be implemented by solving the **OUT OF SCOPE** condition in a later ticket
